Index: src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import './App.css';\r\nimport Users from \"./components/users\";\r\nimport React from \"react\";\r\nimport Hello from \"./components/hellotext\";\r\nimport UserById from \"./components/getBy_ID_feature\";\r\n\r\nfunction App() {\r\n\r\n    return (\r\n\r\n        <div className=\"App\">\r\n\r\n            <header className=\"App-header\">\r\n\r\n                <Users/>\r\n                {/*<UserById/>*/}\r\n                {/*<Hello/>*/}\r\n            </header>\r\n\r\n        </div>\r\n    );\r\n}\r\n\r\nexport default App;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/App.js b/src/App.js
--- a/src/App.js	(revision d9fdddca2f5c5b305b4aea8a3bca55d8bbdf7fab)
+++ b/src/App.js	(date 1609763131701)
@@ -1,24 +1,18 @@
-import './App.css';
+import "./App.css";
 import Users from "./components/users";
 import React from "react";
-import Hello from "./components/hellotext";
-import UserById from "./components/getBy_ID_feature";
 
 function App() {
-
-    return (
-
-        <div className="App">
-
-            <header className="App-header">
-
-                <Users/>
-                {/*<UserById/>*/}
-                {/*<Hello/>*/}
-            </header>
-
-        </div>
-    );
+  return (
+   
+      <div className="App">
+        <header className="App-header">
+         <Users/>
+         {/*<UserById/>*/}
+        </header>
+      </div>
+  
+  );
 }
 
 export default App;
